swagger/v1/location.yaml

```diff
@@ -97,6 +97,13 @@ Location:
           type: boolean
           example: false
           description: Indicates that the location is a Youth Offenders Institute (note that some locations may have mixed populations so have a differing location type).
+        created_at:
+          oneOf:
+          - type: string
+            format: date-time
+          - type: 'null'
+          example: '2017-07-21T17:32:28Z'
+          description: The date-time at which the location was created, in ISO-8601 format
```

Comment:
Might be better to say "The date-time at which the location was imported into Book a Secure Move, in ISO-8601 format" - currently locations are created in NOMIS but not imported into our service until we run the sync